mobile/justfile

```diff
@@ -0,0 +1,71 @@
+# To use this file, install Just: cargo install just
```

Comment:
🙃 Was it on purpose that the `justfile` is in the mobile subfolder; would it be possible to put the `justfile` into the top level project folder? 
Otherwise we should probably be more clear that one has to execute the commands in the `modbile` subfolder.

Reply:
For this PR id made sense but I agree, once we have both projects (#32 ) than I would move this one folder up.